.github/workflows/php-compatibility.yml

```diff
@@ -0,0 +1,18 @@
+name: PHP Compatibility
+
+on:
+  pull_request
+
+jobs:
+  # Check for version-specific PHP compatibility
+  php-compatibility:
+    name: PHP Compatibility
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
```

Comment:
Unrelated but does it make sense to fork this and keep the things we need?

Reply:
It's just a tool that we use to setup PHP, and it works. We could fork it and strip the stuff we don't need, but I don't see any meaningful reason to do so. Did you have something in mind?

Reply:
Not really, the repository name just stood out since it was out of org but I guess it is fine to have it here.